modules/calibration/data/dev_kit_advanced/velodyne_dag/velodyne16.dag

```diff
@@ -0,0 +1,41 @@
+# Define all coms in DAG streaming.
+module_config {
+    module_library : "/apollo/bazel-bin/modules/drivers/lidar_surestar/libsurestar_driver_component.so"
+
+    components {
+      class_name : "CompSureStarDriver"
+      config {
+        name : "velodyne_driver"
+        config_file_path : "/apollo/modules/drivers/velodyne/conf/surestar.conf"
```

Comment:
config_file_path should be "/apollo/modules/drivers/lidar/conf/surestar.conf"  , and same for other lidar conf files